omr/startup/makefile

```diff
@@ -44,7 +44,7 @@ ifeq (1,$(OMR_GC_SEGREGATED_HEAP))
 endif
 
 ifeq (linux,$(OMR_HOST_OS))
-  ifeq (x86,$(OMR_HOST_ARCH))
+  ifneq (,$(filter x86 riscv,$(OMR_HOST_ARCH)))
```

Comment:
I think we can leave out this flag for unrolling loops.

Reply:
Removed for now.